supervisor/addons/model.py

```diff
@@ -578,6 +578,16 @@ def with_journald(self) -> bool:
         """Return True if the add-on accesses the system journal."""
         return self.data[ATTR_JOURNALD]
 
+    @property
+    def signed(self) -> bool:
+        """Return True if the image is signed."""
+        return ATTR_CODENOTARY in self.data
```

Comment:
```suggestion
        return any(service is not None for service in (self.codenotary,))
```
Then when we get more services we just add them to the tuple here

Reply:
yeah, it will be not the only part that have to change then :D